data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
+			branch rich
+				"net worth" >= 4000000000
+			
+			choice
+				`	"What kind of market opportunity?"`
+					goto opportunity
+				`	"Sorry, but I don't have the time to talk about this."`
+					goto walk
+			
+			label rich
+			choice
+				`	"What kind of market opportunity?"`
+					goto opportunity
+				`	"You're only three billion credits? Why shouldn't I pursue your opportunity on my own?"`
+				`	"Sorry, but I don't have the time to talk about this."`
+					goto walk
+			
+			`	"Of course you're worth more than me, Captain <last>, but you are no business owner. I have the connections and skills, but a name like yours will always help.`
+				goto opportunity
+			
+			label walk
+			`	You try to walk away, but Turner stops you by grabbing your arm and continues to talk.`
+			
+			label opportunity
+			`	"Allow me to explain. The journey to the Hai can be a dangerous one, and you know that. I heard many stories during the war of merchants trying to reach the Hai, but thanks to the Navy being distracted, those merchants fell victim to pirates patrolling the systems of the Far North. Now that many merchants are here, they're trapped. They expended most of if not all of their ammunition trying to get here, and now they have none to defend them on their way back. Human ammunition has become a scarce commodity among human captains in this region of space, leaving open a market that's just waiting to be cornered.`
+			`	"I'd like you, Captain <last>, hero of humanity, to help me set up an outfitter here on <origin>, an outfitter that would supply these desperate merchants with relief by supplying them with the ammunition they need, along with outfits they might want to buy for their old ships. I've already received approval from the Hai government for the construction of an outfitter, and with a name like yours backing up this business, the profits are sure to be phenomenal, profits that you would be sharing in by being a founding member of this outfitter."`
+			choice
+				`	"This sounds like a good idea to me. What do you need me to do?"`
+					goto interested
+				`	"Sorry, but I'm not interested in your proposition."`
+			
+			`	"Nonsense, Captain! All you'd need to do is help escort a few freighters from <planet> to Follower and back to here, and for so simple a job you could be receiving enough money to not work another day in your life, or perhaps even support a massive fleet."`
+			choice
+				`	"Again, I'm not interested. Goodbye."`
+				`	"Alright, you have my attention."`
+					goto interested
+			
+			`	"Well, I tried, Captain," Turner says with a smirk. "I'll just have to travel back to the Paradise Worlds and make my money in other areas of business, then." He hands you another business card and winks. "Goodbye, Captain <last>. You should know where to find me if you need me."`
```

Comment:
Something tells me he would want to pursue such a lucrative business opportunity with someone else rather than just abandon it. Even if not, he might _say_ that just to make you feel bad or try to get you to reconsider.

Reply:
The problem with that is that then the shipyard and outfitter still need to be built, even without the player's interaction. I might handle that later, but at least right now I feel that refusing the opportunity to have a human outfitter further north (which could help resupply a player during the Wanderer campaign if they're still using some human weapons) should hold some weight.

Reply:
In that case, maybe we need to come up with a more compelling reason why it can't happen without you. Maybe the Hai authorities don't grant him the license without your blessing, seeing as how you're an upstanding representative of humankind who they feel like they can trust.

@tehhowch Oops, will definitely use the review functionality next time.

Reply:
How about I change this bit to something like
>He hands you another business card and winks. "Goodbye, Captain <last>. You should know where to find me if you need me. Perhaps we can pursue this venture together in the future."

This way there's still the possibility of the shipyard and outfitter being built in the future, just not right now. Turner could certainly build the shipyard and outfitter whenever he wants without the player, but he's trying to get the player involved for the ability to advertise with the player's name, something that could be priceless. (I'm sure there are other "celebrities" that Turner could use, but who better to be associated with than someone who "saved humanity"?)